app/client/components/OverlayLoader.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { css } from "@emotion/core";
+import { textSans } from "@guardian/src-foundations/typography";
+import { Spinner } from "./spinner";
+
+interface OverlayLoaderProps {
+    message: string;
+}
+
+const overlay = css`
+    z-index: 10000;
+	display: block;
+	border: 0 none transparent;
+	overflow-x: hidden;
+	overflow-y: auto;
+	visibility: visible;
+	margin: 0;
+	padding: 0;
+	-webkit-tap-highlight-color: transparent;
+	position: fixed;
+	left: 0;
+	top: 0;
+	width: 100%;
+	height: 100%;
+	text-align: center;
+    ${textSans.medium()};
+`
```

Comment:
It's weird that the pre commit hook didn't pretty this ... maybe `pretty-quick` doesn't know what to do with the template literal string :s

Reply:
as an update to this. We needed to update the version of pretty-quick after bumping our node version